handbook/engineering.md

```diff
@@ -243,9 +243,9 @@ This resulted in basically a full host_software table scan per each software row
 
 ### On constantly changing data
 
-A very complex thing to do well shows presence data in a somewhat real time fashion. In the case of Fleet, that is host seen_time which is what we use to define if a host is online or offline.
+A very complex thing to do well shows presence data in a somewhat real-time fashion. In the case of Fleet, that is host seen_time which is what we use to define if a host is online or offline.
```

Comment:
I'm not sure what this means. Please rewrite for clarity.

Reply:
@zwass, could you look at line 246 and help me decipher how to rewrite it for clarity?

Reply:
```suggestion
It can be difficult to show real-time presence data. For Fleet, that is the host `seen_time` -- the time a host last connected to the server -- which is used to determine whether a host is online.
```